service/history/service.go

```diff
@@ -184,7 +184,6 @@ func (s *Service) Stop() {
 	remainingTime = s.sleep(shardOwnershipTransferDelay, remainingTime)
 
 	s.GetLogger().Info("ShutdownHandler: No longer taking rpc requests")
-	s.handler.PrepareToStop()
```

Comment:
Why?  I thought this is for graceful shutdown.

Reply:
the status is already updated: https://github.com/temporalio/temporal/pull/1037/files/5ca23ff1a2b3f42be78d69b590b55644feebf087#diff-162efb4a3898151e1b88f08e0caacacc43a8212abda815891ac443069b534852R154